tests/acceptance/helpscount-beacon-test.js

```
@@ -0,0 +1,53 @@
+
+    assert.ok(catalogPage.helpscoutBeacon.isPresent, 'Helpscout Beacon is present');
+  });
+
+  test("beacon doesn't render on concepts page", async function (assert) {
+    testScenario(this.server);
+    await conceptsPage.visit();
+
+    assert.false(conceptsPage.helpscoutBeacon.isPresent, 'Helpscout Beacon is not present');
+  });
+
+  test("beacon doesn't render on concepts page", async function (assert) {
```

ESLint

[error] Test name is used on line 31 in the same module.

(qunit/no-identical-names)